brave/browser-laptop
# Issue #9053: Making country field in autofill a dropdown list instead of text input box.

Fixes #9053

I mistakenly closed #10300 opened by @dfperry5. I picked up the 4 commits from the PR and added a commit to add `CommonFormFullWidthDropdown`. Apologies to @dfperry5.

## Submitter Checklist:


## Test Plan:
1. Open `about:autofill`
2. Click `Add Address`
3. Click `Country`
4. Input `united states`
5. Make sure `United States` is selected
6. Hit enter key
7. Click `Save`
8. Open https://www.roboform.com/filling-test-all-fields
9. Select `Country`
10. Make sure `United States` appears on the candidate list

## Automated test plan:
`npm run test -- --grep="Autofill"`

## Reviewer Checklist:

Tests
